docs/_static/_sphinx_javascript_frameworks_compat.js

```diff
@@ -0,0 +1,123 @@
+/* Compatability shim for jQuery and underscores.js.
+ *
+ * Copyright Sphinx contributors
+ * Released under the two clause BSD licence
+ */
+
+/**
+ * small helper function to urldecode strings
+ *
+ * See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/decodeURIComponent#Decoding_query_parameters_from_a_URL
+ */
+jQuery.urldecode = function(x) {
```

Comment:
minor: 

```
jQuery.urldecode = function(x) {
    return x ? decodeURIComponent(x.replace(/\+/g, ' ')) : x;
};
```

Reply:
I copied this Javascript file from somewhere and don't think it is worthwhile changing it (and we would need to change it in all repos for consistency).